content/en/continuous_integration/pipelines/azure.md

```diff
@@ -99,25 +100,31 @@ Example for enabling the hooks in specified projects:
     projectName1 projectName2
 ```
 
+The script supports environment variables `DD_API_KEY` and `DD_SITE`, and flags parameters `--dd-api-key` and `--dd-site`.
```

Comment:
I think it was more useful before the scripts example, readers could miss is so down below

Reply:
```suggestion
```